client/my-sites/stats/site.jsx

```diff
@@ -241,7 +241,12 @@ export default connect(
 		const isJetpack = isJetpackSite( state, siteId );
 		return {
 			isJetpack,
-			hasPodcasts: getSiteOption( state, siteId, 'podcasting_archive' ),
+			hasPodcasts:
+				// Podcasting category slug
+				// TODO: remove when settings API is updated for new option
+				!! getSiteOption( state, siteId, 'podcasting_archive' ) ||
+				// Podcasting category ID
+				!! getSiteOption( state, siteId, 'podcasting_category_id' ),
```

Comment:
https://github.com/Automattic/wp-calypso/pull/23812/files#r178950830 → and here is the other.  I think the right thing to do for both of these cases is to query _both_ options, then remove the old `podcasting_archive` when no more sites are writing to this setting.